frame/contracts/src/wasm/runtime.rs

```diff
@@ -407,68 +457,72 @@ define_env!(Env, <E: Ext>,
 		Ok(())
 	},
 
-	// Retrieve the value under the given key from the storage and return 0.
-	// If there is no entry under the given key then this function will return 1 and
-	// clear the scratch buffer.
+	// Retrieve the value under the given key from storage.
+	//
+	// # Parameters
+	//
+	// - `key_ptr`: pointer into the linear memory where the key of the requested value is placed.
+	// - `out_ptr`: pointer to the linear memory where the value is written to.
+	// - `out_len_ptr`: in-out pointer into linear memory where the buffer length
+	//	is read from and the value length is written to.
```

Comment:
Can we replace a tab with spaces for aligning purposes? This is because using tabs for alignment in comments doesn't produce the best results because it is seemingly interpreted differently by different editors.

Reply:
Yes. It is now aligned by spaces.